pypolestar/polestar.py

```diff
@@ -109,9 +108,11 @@ def get_car_information(self, vin: str) -> CarInformationData | None:
             KeyError: If the VIN doesn't exist
             ValueError: If data conversion fails
         """
+
         if vin not in self.available_vins:
             raise KeyError(vin)
-        if data := self.data_by_vin[vin].get(CAR_INFO_DATA, {}).get("data"):
+
+        if data := self.data_by_vin[vin].get(CAR_INFO_DATA):
```

Comment:
maybe test if data is not None instead of relying on the exception ? 

Furthermore the error message could use the VIN number in order to assist in debugging

Reply:
Perhaps something like 0cb938309d19a91fb5618aa949ded7581cbcc6ec  ?

Reply:
That is great! But I was more concerned with the get() which might return None

Reply:
Do you think we should raise an exception if no data is available instead of returning None?

Reply:
Unfortunately the API is down more then we would expect. Returning None is fine, and should be processed further ahead in a retry logic.